client/src/plus/collections/collection-list-item.tsx

```diff
@@ -76,7 +76,7 @@ export function CollectionListItem({
             />
             <DropdownMenu>
               <ul className="dropdown-list" id="collection-item-dropdown">
-                {"id" in item && onEditSubmit && (
+                {"notes" in item && onEditSubmit && (
```

Comment:
To avoid this confusion, maybe it would be better renaming `FrequentlyViewedEntry.{id => index}`.

Reply:
Thanks!
Done in `id -> index` commit.